libs/zzz/db/src/Database/DataManagers/CharacterOptManager.ts

```diff
@@ -98,23 +179,45 @@ export class CharacterOptManager extends DataManager<
       optConfigId,
     } = obj as CharOpt
     if (!Array.isArray(conditionals)) conditionals = []
-    const hashList: string[] = [] // a hash to ensure sheet:condKey:src:dst is unique
-    if (!targetSheet || !targetName || !getFormula(targetSheet, targetName)) {
-      targetSheet = undefined
-      targetName = undefined
+    // Validate target for formula
+    if (target?.name) {
+      const formula = getFormula(target)
+      if (formula) {
+        let { damageType1, damageType2 } = target
+        if (formula.name !== 'standardDmgInst') {
+          damageType1 = undefined
```

Comment:
removing anomaly from anomalyDmgInst?

Reply:
It's only the database representation, having it `undefined` means it won't overwrite `anomalyDmgInst.damageType1`